ios/FluentUI/Navigation/UINavigationItem+Navigation.swift

```diff
@@ -145,3 +145,14 @@ import UIKit
         }
     }
 }
+
+extension UINavigationItem {
+    var canBeCompact: Bool {
+        // Large leading titles already do the right thing in compact nav bars
+        if titleStyle == .largeLeading {
+            return true
+        }
+
+        return subtitle == nil && titleStyle != .largeLeading && accessoryView == nil
```

Comment:
Do we need to check for the `titleStyle` being something other than `.largeLeading` here? We're already returning above if it is `.largeLeading`, so it should have to be something else to get to this point.

Reply:
This was a remnant of when I was trying to figure out the correct logic to use.